### ISSUE TITLE
CI failed when install hypercontainer

### ISSUE THREAD

**Issue by resouer** [kubernetes/frakti#250]
https://travis-ci.org/kubernetes/frakti/jobs/289321878

```
Making all in cmd
make[1]: Entering directory `/tmp/src/github.com/hyperhq/hyperd/cmd'
Making all in hyperd
make[2]: Entering directory `/tmp/src/github.com/hyperhq/hyperd/cmd/hyperd'
go build -tags "static_build  with_libvirt btrfs_noversion libdm_no_deferred_remove" -ldflags "-X github.com/hyperhq/hyperd/utils.VERSION=0.8.1 -X github.com/hyperhq/hyperd/utils.GITCOMMIT=`git describe --dirty --always --tags 2> /dev/null || true`" hyperd.go
make[2]: Leaving directory `/tmp/src/github.com/hyperhq/hyperd/cmd/hyperd'
Making all in hyperctl
make[2]: Entering directory `/tmp/src/github.com/hyperhq/hyperd/cmd/hyperctl'
go build -ldflags "-X github.com/hyperhq/hyperd/utils.VERSION=0.8.1 -X github.com/hyperhq/hyperd/utils.GITCOMMIT=`git describe --dirty --always --tags 2> /dev/null || true`" hyperctl.go
make[2]: Leaving directory `/tmp/src/github.com/hyperhq/hyperd/cmd/hyperctl'
Making all in vmlogd
make[2]: Entering directory `/tmp/src/github.com/hyperhq/hyperd/cmd/vmlogd'
go build -ldflags "-X github.com/hyperhq/hyperd/utils.VERSION=0.8.1 -X github.com/hyperhq/hyperd/utils.GITCOMMIT=`git describe --dirty --always --tags 2> /dev/null || true`" vmlogd.go
make[2]: Leaving directory `/tmp/src/github.com/hyperhq/hyperd/cmd/vmlogd'
make[2]: Entering directory `/tmp/src/github.com/hyperhq/hyperd/cmd'
make[2]: Nothing to be done for `all-am'.
make[2]: Leaving directory `/tmp/src/github.com/hyperhq/hyperd/cmd'
make[1]: Leaving directory `/tmp/src/github.com/hyperhq/hyperd/cmd'
make[1]: Entering directory `/tmp/src/github.com/hyperhq/hyperd'
make[1]: Nothing to be done for `all-am'.
make[1]: Leaving directory `/tmp/src/github.com/hyperhq/hyperd'
!!! Error in /home/travis/gopath/src/k8s.io/frakti/hack/lib/hyper.sh:75
  'return 1' exited with status 1
Call stack:
  1: /home/travis/gopath/src/k8s.io/frakti/hack/lib/hyper.sh:75 frakti::hyper::install_hypercontainer(...)
  2: hack/install-hypercontainer.sh:23 main(...)
Exiting with status 1
!!! Error in /home/travis/gopath/src/k8s.io/frakti/hack/lib/hyper.sh:21
  '( export GOPATH=${HYPERD_TEMP}; mkdir -p "${GO_HYPERHQ_PATH}"; frakti::log::status "install necessary tools"; frakti::hyper::preinstall; frakti::log::status "build hyperd"; frakti::hyper::build_hyperd; frakti::log::status "build hyperstart"; frakti::hyper::build_hyperstart )' exited with status 1
Call stack:
  1: /home/travis/gopath/src/k8s.io/frakti/hack/lib/hyper.sh:21 frakti::hyper::install_hypercontainer(...)
  2: hack/install-hypercontainer.sh:23 main(...)
Exiting with status 1
The command "hack/install-hypercontainer.sh" failed and exited with 1 during .
```

**Comment by resouer** [comment]
happened in https://github.com/kubernetes/frakti/pull/249

**Comment by resouer** [comment]
Confirmed locally.

cc @bergwolf would you like to give this a fix?  It seems like `./hack/install-hypercontainer.sh ` need to be updated.

**Comment by bergwolf** [comment]
ok, I'll take this one.

**Comment by resouer** [comment]
fixed by #255
